charts/spire/charts/spire-server/templates/configmap.yaml

```diff
@@ -80,6 +80,18 @@ data:
       {{- end }}
       {{- end }}
 
+      {{- with .Values.upstreamAuthority.spire }}
+      {{- if eq (.enabled | toString) "true" }}
+      UpstreamAuthority "spire" {
+        plugin_data {
+          server_address      = {{ .server.address | quote }}
+          server_port         = {{ .server.port }}
+          workload_api_socket = "/run/spire/upstream_agent/spire-agent.sock"
```

Comment:
Should we hardcode this or also make variable? Looks like it should be a variable as it depends how you deploy the agent.

Reply:
We could make it variable to match the upstream csi driver deployment, yeah. In theory the upstream driver deployment is only used for this one workload though, so changing it in the upstream / making it configurable here doesn't make too much sense to me?

Reply:
Probably I'm confused due to the 3 helm installs.